lib/fog/libvirt/models/compute/templates/server.xml.erb

```diff
@@ -94,6 +94,16 @@
       <target type='virtio' name='org.qemu.guest_agent.0'/>
     </channel>
 <% end -%>
+    <rng model='virtio'>
+<%
+  rng_backend_model = virtio_rng[:backend_model] ? virtio_rng[:backend_model] : 'random'
+-%>
+<% if virtio_rng[:backend_path] -%>
```

Comment:
I think it would be a little better to ask on `virtio_rng[:backend_model]` and if provided we include the `<rng>` tag with either backend/path (if both are provided) or just the backend with default path. 

I am saying that to not introduce `rng` automatically for all with the next release.

Reply:
> I am saying that to not introduce `rng` automatically for all with the next release.

I naively assume that enabling `virtio_rng` is a useful default setting for people not spending any thoughts on random numbers. Therefore I would like to gain a better understanding of the reasons for your concerns.

Reply:
@strzibny I tend to agree with @wiene here: Having to virtio-rng inside a VM is unexpected to those aware of the problem to get entropy inside a virtual environment, and harmful to those who are unaware, since they are subject to security and / or performance issues. 
So I would vote for {{rng}} to be added and "on by default", if that's not a too significant change for the next release. Thoughts?

Reply:
I am actually not against it. My comment was purely based on selfish maintainership issue of me spending time now to find out if it's okay as a default :). As a non-default I would already merge it. If anything it provoked a response from @olifre which gives me confidence in this being a default.

Reply:
> My comment was purely based on selfish maintainership issue of me spending time now to find out if it's okay as a default :)

That's fine ;-). Better ask for more reasoning early than be sorry later. 

> If anything it provoked a response from @olifre which gives me confidence in this being a default.

So that means I've convinced you that the current proposal is fine? Then I would go ahead and give it some testing and report back.

Reply:
Yes.

Reply:
Thanks! 
I gave it sufficient testing now - trying different ``backend_path``s for ``backend_model`` ``random``, also trying ``backend_model`` ``rng-builtin`` (which expectedly caused an error message by our ``libvirtd`` since it does not support that yet). It seems to work as planned for us :-).

Reply:
@olifre thanks a lot for the testing, I will merge now:)

Reply:
@strzibny Thanks, and thanks for the helpful discussion, too :+1:.